keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -645,10 +653,14 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Give a driver that does not ControllerModifyVolume, CSI volume should not be modified.
 - If ControllerModifyVolume fails, PVC should have appropriate events.
 
+[Tests PR](https://github.com/kubernetes/kubernetes/pull/121849)
+
+k8s-triage TODO
```

Comment:
What does this mean?

Reply:
Updated TODO placeholder for "Waiting until PR merged" placeholder.